tfe-scripts/tf-update-variable-values.sh

```diff
@@ -20,14 +20,61 @@ else
     jq -r '.data.id'))
 fi
 
+update_value () {
+  ./tfe-scripts/tf-update-variable-value.sh "$WORKSPACE_ID" "$1" "$2"
+}
+
 while IFS= read -r line; do
-  VAR_KEY="$(cut -d'=' -f1 <<<"$line")"
-  VAR_VAL="$(cut -d'=' -f2- <<<"$line")"
+  var_key="$(cut -d'=' -f1 <<<"$line")"
+  var_val="$(cut -d'=' -f2- <<<"$line")"
+
+  if [ ! -z "$var_key" ] && [ ! -z "$var_val" ]; then
+    if [ $var_key == "namespace_apps" ]; then
+      # create double quote escaped namespace_apps array string and collect namespaces
```

Comment:
Let's make sure we lint this and all other bash scripts with bats? Quoting is hard.
Example: https://github.com/bcgov/cas-pipeline/blob/master/.circleci/config.yml#L16

Reply:
I will see how I could add Bats tests here

Reply:
sorry, silly me. Bats is our unit testing harness and Shellcheck is our bash linter. https://github.com/bcgov/cas-pipeline/blob/master/.circleci/config.yml#L11

Would be super ultra cool if we unit tested these scripts too ;)